src/ray/raylet/worker_pool_test.cc

```diff
@@ -78,19 +78,9 @@ class WorkerPoolTest : public ::testing::Test {
 
   std::shared_ptr<Worker> CreateWorker(pid_t pid,
                                        const Language &language = Language::PYTHON) {
-    std::function<void(LocalClientConnection &)> client_handler =
-        [this](LocalClientConnection &client) { HandleNewClient(client); };
-    std::function<void(std::shared_ptr<LocalClientConnection>, int64_t, const uint8_t *)>
-        message_handler = [this](std::shared_ptr<LocalClientConnection> client,
-                                 int64_t message_type, const uint8_t *message) {
-          HandleMessage(client, message_type, message);
-        };
-    boost::asio::local::stream_protocol::socket socket(io_service_);
-    auto client =
-        LocalClientConnection::Create(client_handler, message_handler, std::move(socket),
-                                      "worker", {}, error_message_type_);
+    WorkerID worker_id = WorkerID::FromRandom();
     return std::shared_ptr<Worker>(
-        new Worker(pid, language, -1, client, client_call_manager_));
+        new Worker(worker_id, pid, -1, language, client_call_manager_));
```

Comment:
Should have already been there, but inline comment for what the -1 means please.